vendor/github.com/Microsoft/go-winio/README.md

```diff
@@ -9,7 +9,7 @@ to reuse the thread to schedule another goroutine. This limits support to Window
 newer operating systems. This is similar to the implementation of network sockets in Go's net
 package.
 
-Please see the LICENSE file for licensing information.
+See the LICENSE file for licensing information.
```

Comment:
Oh! Files inside the `vendor/` directory are dependencies we use, but those are not maintained by us, and none of the files inside the `vendor/` directory should be edited by us.

Can you revert changes inside `vendor/` ?